simon_says/ui/src/config.js

```diff
@@ -0,0 +1,63 @@
+export const rosEndpoint = window.rosEndpoint || process.env.REACT_APP_ROSBRIDGE_WS || 'ws:0.0.0.0:9090/';
```

Comment:
Might be useful to inform in Readme that the development version will listen to 'ws:localhost:9090'

Reply:
it does not say localhost here